.verilog_format

```diff
@@ -0,0 +1,26 @@
+--column_limit=100
+--indentation_spaces=2
+--line_break_penalty=2
+--over_column_limit_penalty=10000
+--wrap_spaces=4
+--assignment_statement_alignment=align
+--case_items_alignment=align
+--class_member_variable_alignment=align
+--distribution_items_alignment=align
+--enum_assignment_statement_alignment=align
+--formal_parameters_alignment=align
+--formal_parameters_indentation=indent
+--module_net_variable_alignment=align
+--named_parameter_alignment=align
+--named_parameter_indentation=indent
+--named_port_alignment=align
+--named_port_indentation=indent
+--port_declarations_alignment=align
+--port_declarations_indentation=indent
```

Comment:
The lowrisc style guide generally encourages tabular alignment for "things"; I think if you switch over to tabular alignment for named_*_indentation, I would also suggest to `--port_declarations_indentation=align`

Reply:
`ERROR: Illegal value 'align' specified for flag 'port_declarations_indentation'; Invalid IndentationStyle: 'align'`
I will stick to indent for now.